### ISSUE TITLE
Increase batch_size to more than 1

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#87]
Currently it is just one, which I think is hurting our performance as it leads to very noisy gradients. Should have just a quick chat about what changes to the loss function and/or dataset classes are needed.

**Comment by manuelli** [comment]
Don't think batch size 1 is a problem, may actually be a good thing
